03_letters/srini/main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Add `TestLetters` and `TestSortLetters`
Include corner cases (non ascii unicode, empty strings etc.)

Reply:
Added few more test cases